README.md

```diff
@@ -1,38 +1,69 @@
-# Blog flask extension
+# Ubuntu blog extension
```

Comment:
I don't quite agree with this characterisation. This is not just for ubuntu.com/blog, but also for snapcraft.io/blog, jaas.ai/blog (as you know) etc.. And it's quite possible that we might in the future make it so snapcraft blog posts don't show up at ubuntu.com/blog at all.

I consider ubuntu.com/blog to be just one implementation of this module.

It is, however, currently inherently true that this blog module gets its data from the Wordpress at admin.insights.ubuntu.com. So I wouldn't mind mentioning that fact in the description somewhere. However, it does seem neatest to me to continue to describe the module itself as simply "Blog" or maybe "Canonical blog".

Reply:
This isn't attempting to describe where this module is used, but the hard-coded feed it will output. 

When I first came to this module, I had no idea the feed was hard-coded and spent quite a while trying to figure out how to pass it a JSON endpoint. The feed comes from `https://admin.insights.ubuntu.com`, the front-end of which used to be `https://insights.ubuntu.com` which now redirects to `https://ubuntu.com/blog` - therefore it's not a generic extension to output a blog feed, it can only output posts from the ubuntu blog wordpress installation, therefore that should be made clear immediately in the title.

Reply:
We can call it "Canonical blog" if you like.

Regardless of its history, what it actually is is a plugin that pulls content from a CMS API back-end that is the only place Canonical as a company publish blog posts to, whether those blog posts be about Ubuntu, Snapcraft, Jaas or anything else that Canonical get up to. And it's not a place that the Ubuntu community can publish to. So it isn't "Ubuntu blog".

Reply:
Done